html/index.html

```diff
@@ -4,9 +4,13 @@
         <meta charset="utf-8">
         <title>VR Party</title>
         <link rel="stylesheet" href="css/style.css" type="text/css">
+        <link rel="stylesheet" href="css/jquery-ui.min.css" type="text/css">
+        <link rel="stylesheet" href="css/jquery-ui.theme.min.css" type="text/css">

```

Comment:
https://code.jquery.com/ui/1.11.4/themes/flick/jquery-ui.css
https://code.jquery.com/ui/1.11.4/jquery-ui.min.js

should do the trick